### ISSUE TITLE
Bug in `nonzero_idx`?

### ISSUE THREAD

**Issue by krstopro** [atksh/annax#14]
Is it me, or the following two lines should be swapped?
https://github.com/atksh/annax/blob/566aa3e1efe80d619ca00e93b9e3c7c43cc3b6a9/src/annax/ops.py#L51-L52

**Comment by webbrain-one** [comment]
Good catch! 👀 I've opened PR #17 to swap those lines. Thanks for pointing this out!
